llvm/test/CodeGen/AMDGPU/vgpr-remat.mir

```diff
@@ -46,3 +46,50 @@ body: |
     %4.sub1:vreg_96 = COPY %2:vgpr_32
     S_ENDPGM 0, implicit %4
 ...
+
+---
```

Comment:
The test is running with -mcpu=gfx900, which doesn't have v_pk_mov_b32. At one point I was working on making MIR tests less permissive about mismatches, maybe should adjust the target-cpu for the run line